backbone.js

```diff
@@ -1140,6 +1140,7 @@
         removed.push(model);
         this._removeReference(model, options);
       }
+      if (models.length > 0 && !options.silent) delete options.index;
```

Comment:
Let's just delete the prop to make it easy. But why are we guarding with `this.models > 0`?